docs/GWDM/1.0.form.json

```diff
@@ -153,7 +153,12 @@
                   "description": "Please provide a list of relevant and specific keywords that can improve the search engine optimisation (SEO) of your dataset as a comma separated list. Notes: Onboarding portal will suggest keywords based on title, abstract and description. We are compiling a standardised list of keywords and synonyms across datasets to make filtering easier for users.",
                   "guidance": "- Please provide **relevant** and **specific keywords** that can **improve the search engine optimization** of your dataset.\\n- Please **enter one keyword at a time** and click **Add New Field** to add further keywords.\\n- Text from the title is automatically included in the search, there is no need to include this in the keywords.\\n- Include words that researcher may include in their searches.",
                   "examples": [
-                        "Health Data, Research, SAIL, Primary care, GP"
+                        "Outpatient Care",
```

Comment:
The examples are used as placeholders on the metadata onboarding form. I'm not sure how a multi-line example will appear on the FE. Might be worth checking with @philreekshdr

Reply:
Hi @BSnelling, thanks for flagging, i'll drop Phil a message once it's merged, I think there are quite a few examples which will have an array, will see how this is handled